package.json

```diff
@@ -12,15 +12,13 @@
   "dependencies": {
     "@kiltprotocol/sdk-js": "^0.32.0",
     "@kiltprotocol/types": "^0.32.0",
-    "@kiltprotocol/utils": "^0.32.0",
     "@kiltprotocol/vc-export": "^0.32.0",
-    "@polkadot/api": "^9.4.2",
     "@polkadot/util": "^10.1.9",
-    "@polkadot/util-crypto": "^10.1.9",
-    "react": "^18.2.0",
-    "ts-node": "^10.9.1"
+    "valid-url": "^1.0.9"
   },
   "devDependencies": {
+    "@kiltprotocol/utils": "^0.32.0",
```

Comment:
If the full SDK package is a dependency anyway, does it still make sense to import from this one directly?